### ISSUE TITLE
eukccdb

### ISSUE THREAD

**Issue by housw** [Finn-Lab/EukCC#2]
Hi, 

how can I get an up to date `eukccdb`? The EBI one seems doesn't work on our server:
```
01/24/2020 16:51:38:  Starting EukCC
01/24/2020 16:51:38:  This is an old database, it does not profide machine readable information about the database
01/24/2020 16:51:38:  Running GeneMark-ES
01/24/2020 16:51:49:  GeneMark-ES failed on this bin
```

Thanks, 
Shengwei

**Comment by openpaul** [comment]
Hi, yes we just noticed this as well. The database will be updated shortly, but EukCC should still work with this old version of the DB.
GeneMark-ES seems to have failed independently in your case, so proteins could not be predicted. 
I will close this issue as soon as we pushed the new database online.

**Comment by openpaul** [comment]
This should be fixed with db version 1.1: 

http://ftp.ebi.ac.uk/pub/databases/metagenomics/eukcc/eukcc_db_v1.1.tar.gz
